src/components/implementation/tests/sqlite_tests/sqlite_tests.c

```diff
@@ -0,0 +1,44 @@
+#include <cos_component.h>
+#include <llprint.h>
+#include <sqlite3.h>
+#include <cos_defkernel_api.h>
+
+void
+cos_init(void)
+{
+}
+
+int result_print(void *v, int argc, char **argv, 
```

Comment:
your line width wraparound seems too aggressive. I don't think that the argument needs to be on another line.